st2common/tests/unit/test_action_alias_utils.py

```diff
@@ -230,3 +233,28 @@ def test_ending_parameters_matching(self):
         parser = ActionAliasFormatParser(alias_format, param_stream)
         extracted_values = parser.get_extracted_param_value()
         self.assertEqual(extracted_values, {'pony1': 'foo', 'pony2': 'bar'})
+
+
+class TestSearchRegexTokens(TestCase):
+    beginning_tokens = ((AT, AT_BEGINNING), (AT, AT_BEGINNING_STRING))
+    end_tokens = ((AT, AT_END), (AT, AT_END_STRING))
+
+    def test_beginning_tokens(self):
```

Comment:
Should we also have one test case for:

*  parse("asdf") - no ending and beginning tokens - self.assertFalse(search_regex_tokens(self.beginning_tokens, tokens)), self.assertFalse(search_regex_tokens(self.ending_tokens, tokens))

Reply:
Good idea - done!